app/models/event.rb

```diff
@@ -22,6 +22,14 @@ def self.filters_to_json
     ].to_json
   end
 
+  def self.years_to_json
+    events = Event.all.order(start_date: :asc)
+    start_year = events.pluck(:start_date).compact.sort.first.year
+    end_year = events.pluck(:start_date).compact.sort.last.year
```

Comment:
I think it would make more sense to get the `end_date` here instead of `start_date`

Reply:
Ok thanks @Levia I will make that change!